unyt/array.py

```diff
@@ -265,7 +266,12 @@ def _coerce_iterable_units(input_object, registry=None):
                 return unyt_array(np.array(ret), ff, registry=registry)
```

Comment:
While we're at it, might as well reduce the number of return statement to the bare minimum ?
```suggestion
                ret = unyt_array(np.array(ret), ff, registry=registry)
```